core/src/main/java/org/frankframework/stream/document/DocumentBuilderFactory.java

```diff
@@ -38,9 +39,9 @@ public static IDocumentBuilder startDocument(DocumentFormat format, String rootE
 	public static IDocumentBuilder startDocument(DocumentFormat format, String rootElement) throws SAXException {
 		switch (format) {
 		case XML:
-			return new XmlDocumentBuilder(rootElement);
+			return new XmlDocumentBuilder(rootElement, new StringWriter());
 		case JSON:
-			return new JsonDocumentBuilder();
+			return new JsonDocumentBuilder(new StringWriter());
```

Comment:
We need to migrate away from this at some point, for now this is the only place this 'evil' in memory writer is used.